content/guides/destructuring.adoc

```diff
@@ -424,7 +501,9 @@ You can even destructure using auto-resolved keywords, which will again be bound
 
 Creating and destructuring maps with auto-resolved keywords allow us to write code using a namespace alias (here `p`) that is defined by a `require` in the current namespace, giving us a means of namespace indirection that can be changed at a single place in the code.
 
-All symbols bound in the context of destructuring can be further destructured - this allows destructuring to be used in a nested fashion for both sequential and associative destructuring. It is less obvious, but this also extends to the symbol defined after `&`.
+=== Destructuring the `&` seq
+
+Similarly to sequential destructuring, the value (a seq) captured using `&` can be further destructured. 
```

Comment:
Pulling out the subheader is good here, but  I liked the old text better and it seems like you lost some of the intent that made a bigger point.

Reply:
Ok, let's see what I can come up with.